lib/transition.js

```diff
@@ -294,6 +308,24 @@ class Transition {
       }
       return undefined
     }
+
+    /**
+     * Check if transition prop is inside vendor specific rule
+     */
+    inVendorSpec (decl) {
+      let { parent } = decl
+      let { selector, type } = parent
+
+      if (decl._autoprefixerInVendorSpecific !== undefined) {
+        return decl._autoprefixerInVendorSpecific
+      }
+
+      let inSpecific = type === 'rule' && selector.includes(':-') &&
```

Comment:
I think this form will be more readable:

```js
if (parent.type !== 'rule') {
  return false
} else if (!parent.selector.includes(':-')) {
  return false
} else {
  return this.prefixes.add.selectors.find(s => parent.selector.includes(s.name))
}
```

Reply:
Nice approach! Rewrote it.